tox.ini

```
@@ -0,0 +1,20 @@
+[tox]
+envlist = py27,
+          py38,
+minversion = 1.6
+
+[testenv]
+deps = py27: pytest < 5.0
+       py38: pytest
+       pytest-cache
+       pytest-cov
+       pytest-flake8
+       .[test]
```

```suggestion
extras = test
```

This is the new spelling of extras since tox 2.4 (which should be set as minversion than)

---

Done.